pkg/metricscollector/prommetrics.go

```diff
@@ -121,6 +121,16 @@ var (
 		},
 		[]string{"namespace", "type", "resource"},
 	)
+
+	scaledObjectPausedTotal = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_object",
+			Name:      "paused_total",
+			Help:      "Total number of scaled objects currently in a paused state",
```

Comment:
Same remark as for opentelemtry - don't track "total number" but just mark if ScaledObject is paused - take an inspiration from Activity